content/ember-5-0-released.md

```diff
@@ -0,0 +1,331 @@
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.0, please review the [Ember.js 5.0.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.0.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+EmberData 4.12 will remain an actively supported LTS for the entire duration of the 5.x cycle, extending until the first 6.x LTS is released. This is in addition to our regular LTS support policy, and only applies to the EmberData project.
+
+This support extends beyond bug-fixes. If minor enhancements can be made to better support new presentation class implementations that support the 4.x series we will willingly accept them. Our goal is that we want no one left behind.
+
+To learn about the motivation and goals for upcoming changes to EmberData in 5.x,
+read the [blog post, EmberData 5.X Update](https://blog.emberjs.com/ember-data-5-x-update-2023-04-15/).
+<!-- alex ignore retext-equality -->
+This will help you form the mental model of what to expect across the 5.x series,
+and understand deprecation removals in the context of the upcoming goals.
+
+The sections below will provide a high-level summary of what has changed between
+EmberData 4.12 and 5.0.
+
+### Changes in EmberData 5.0
+
+#### Bug Fixes
+
+EmberData 5.0 introduced 7 bug fixes:
+
+- [#8621](https://github.com/emberjs/data/pull/8621) - `normalizeErrorResponse` will provide serialized JSON instead of `[object Object]` if an object is part of the return.
+- [#8598](https://github.com/emberjs/data/pull/8598) - api docs generation should maintain a stable relative path, fixing `Edit on Github` and `Defined in:` links.
+- [#8566](https://github.com/emberjs/data/pull/8566) - previously, an identity notification was being triggered unnecessarily every time a record was saved.
+- [#8597](https://github.com/emberjs/data/pull/8597) - this fix ensures that promise caches are not shared among different fields.
+- [#8594](https://github.com/emberjs/data/pull/8594) - regression that caused the Store class to no longer extend from EmberObject has been fixed.
+- [#8570](https://github.com/emberjs/data/pull/8570) - fixes an issue where the entire RecordArray was being cleared, even if the remaining record did not match the record that was removed.
+- [#8555](https://github.com/emberjs/data/pull/8555) - fixes polymorphic assertions when deprecated code is removed and improves polymorphic developer experience.
+
+#### Features
+
+EmberData 5.0 introduced 0 features.
+
+#### Deprecations
+
+EmberData 5.0 introduced 0 deprecations. It removed all deprecations that were
+introduced before 4.12 and slated for removal in 5.0.
+
+Below we've listed some of the most significant API removals in EmberData 5.0.
+For an exhaustive list of removals, see the [EmberData 4.x deprecation guide](https://deprecations.emberjs.com/ember-data/v4.x) and [#8550](https://github.com/emberjs/data/pull/8550).
+
```

Comment:
may also want to link the api docs for deprecations:

- how to mark deprecations as resolved (removable) in 4.12: https://api.emberjs.com/ember-data/4.12/modules/@ember-data%2Fdeprecations
- list of deprecations https://api.emberjs.com/ember-data/4.12/classes/CurrentDeprecations/properties